AGENT.md

```diff
@@ -12,7 +12,15 @@ This project is the Buildkite CLI (`bk`)
 ## Project Structure
 - Main binary: `cmd/bk/main.go`
 - GraphQL schema: `schema.graphql`
-- CLI commands: `pkg/cmd/`
+- CLI commands: `pkg/cmd/` (legacy Cobra implementation, unused)
```

Comment:
If Cobra is gone, can we get rid of it? Also, with Cobra we had _some_ test coverage – not sure how much we have in place for Kong now? Still going through the PR 👀

Reply:
Gah, I swear I had fixed this… but no, look like there's still some work to do that I missed. Will take a look.

Reply:
Alright, just pushed up what should be the rest